source/content/guides/multisite/01-introduction.md

```diff
@@ -41,17 +41,19 @@ We do not support uses of WordPress Multisite that run functionally-different or
 ## Request a WordPress Multisite
 WordPress Multisite requires a special configuration that is only available to select customers. Access to WordPress Multisite requires a Gold, Platinum, or Diamond level [Workspace plan](https://pantheon.io/plans/pricing).
 
-By default, Pantheon employees must create new Multisites on your behalf. Reach out to your account manager to request site creation for a new WordPress Multisite.
+By default, Pantheon employees must create the WordPress Multisite upstream on your behalf so your team can create new Multisites on the platform. This is because they use the `wordpress_network` [framework](/glossary/#framework) which includes Multisite specific Nginx configurations and differs from the default `wordpress` framework. Reach out to your account manager to request site creation for a new WordPress Multisite.
```

Comment:
```suggestion
Pantheon employees must enable a WordPress Multisite custom upstream on your behalf so your team can create new Multisites self-serve in your professional workspace. This is because WordPress Multisites use the `wordpress_network` [framework](/glossary/#framework), which includes Multisite specific Nginx configurations and differs from the default `wordpress` framework. Reach out to your account manager to get started, or if you don't have an account manager, you can [contact sales](https://pantheon.io/contact-us).
```